prog/plugin/main.go

```diff
@@ -39,51 +46,102 @@ func main() {
 
 	SetLogLevel(logLevel)
 
-	Log.Println("Weave plugin", version, "Command line options:", os.Args)
+	Log.Println("Weave plugin", version, "Command line options:", os.Args[1:])
 
-	dockerClient, err := docker.NewClient("unix:///var/run/docker.sock")
+	dockerClient, err := docker.NewVersionedClient("unix:///var/run/docker.sock", "1.21")
```

Comment:
In the code or in the PR?  I see it as fixing a simple oversight.  There are no choices for specific number at present; this is the only released version that will run the plugin.